hwdb/60-evdev.hwdb

```diff
@@ -269,6 +269,15 @@ evdev:name:AlpsPS/2 ALPS GlidePoint:dmi:*svnLENOVO:*pvrLenovoYoga500-14ISK*
  EVDEV_ABS_35=124:3955:36
  EVDEV_ABS_36=103:1959:26
 
+#########################################
+# Positivo
+#########################################
+
+# Positivo EC10IS1 classmate
+evdev:input:b0003v22B9p0005*
```

Comment:
To be upstreamable, should this instead be a DMI match? I assume the input device IDs are not specific to this product.

Reply:
I can use a DMI. Please note that (I'm looking into this now) we are basically configuring the touchscreen as it was a touchpad. Not sure if upstream they will be pleased.